bin/git-forgit

```diff
@@ -849,10 +849,10 @@ _forgit_checkout_branch() {
     [[ -z "$branch" ]] && return 1
 
     # track the remote branch if possible
-    if [[ "$branch" == "remotes/origin/"* ]]; then
-        if git branch | grep -qw "${branch#remotes/origin/}"; then
+    if [[ "$branch" == "remotes/"* ]]; then
```

Comment:
What happens if two remotes have the same branch?

e.g.
remotes/origin/foo
remotes/upstream/foo

?

Reply:
The one you select gets checked out and tracked as `foo`. This is the same behavior when running `git checkout --track <BRANCH>`.